xla/README.md

```diff
@@ -0,0 +1,124 @@
+# TPU support
+
+This project utilizes [`Fabric`](https://lightning.ai/docs/fabric/stable), which supports TPUs via [PyTorch XLA](https://github.com/pytorch/xla).
+
+To set up a Google Cloud instance with a TPU v4 VM, run the following commands:
```

Comment:
Would it be better to generalize this doc for TPUs as a whole instead of just TPU v4 VM? From what I can see the main change would be to define `version`, `accelerator-type`, `zone` as environment variables that the user defines